runtime/list.go

```diff
@@ -163,6 +163,14 @@ func listAppend(f *Frame, args Args, kwargs KWArgs) (*Object, *BaseException) {
 	return None, nil
 }
 
+func listExtend(f *Frame, args Args, kwargs KWArgs) (*Object, *BaseException) {
+	argc := len(args)
+	if argc != 2 {
+		return nil, f.RaiseType(TypeErrorType, fmt.Sprintf("extend() takes exactly one argument (%d given)", argc-1))
```

Comment:
list.extend() is okay, I did put in test cases, https://github.com/S-YOU/grumpy/blob/b9cea1251a50e636d90d0f546bb62e71eb6c4494/runtime/list_test.go#L247
which returns `extend() takes exactly one argument (0 given)`
because first argument is list itself as far as I noticed.
Is there is case that unbound method can be called?

I am okay to put argc, but error message is one number off by CPython error message, ok to proceed that?